src/poliastro/plotting/orbit/backends/base_backend.py

```diff
@@ -0,0 +1,232 @@
+"""A module implementing the base class for a orbit plotter backend."""
+
+
+class _OrbitPlotterBackend:
```

Comment:
Maybe rename `backends/base_backend.py` to `backends/_base.py` + rename `_OrbitPlotterBackend` to `OrbitPlotterBackend`? (i.e. make the module private, but the class public)

Reply:
Sounds good!